docs/develop/python/message-passing.mdx

```diff
@@ -244,73 +253,92 @@ async def main():
     )
 ```
 
-### Sending an Update {#send-update}
+### Send an Update {#send-update-from-client}
 
-- A client sending an Update must be prepared to wait until the Server has delivered the Update to the Worker: that means that the Worker must not be offline, and must not respond too slowly.
-  If you want the server to send a response as soon as it receives your request, then you must use a Signal instead.
-- `WorkflowExecutionUpdateAccepted` will be added to Event History when the Worker has responded to the Server that the Update passed validation.
-- `WorkflowExecutionUpdateCompleted` will be added to Event History when the Worker has responded to the Server that the Update has completed.
+An Update is a synchronous, blocking call that can change the state of a Workflow Execution and return a response.
+It combines Signal and Query elements with lower overhead and latency.
 
+A client sending an Update must wait until the Server delivers the Update to a Worker.
+Workers must be available and responsive.
+If you need a response as soon as the Server receives the request, use a Signal instead.
+Also note that you can't send Updates to other Workflow Executions or perform an Update equivalent of Signal-With-Start.
 
-It is not yet possible to send an Update to another Workflow, or to do the Update equivalent of Signal-With-Start.
+- `WorkflowExecutionUpdateAccepted` is added to the Event History when the Worker confirms that the Update passed validation.
+- `WorkflowExecutionUpdateCompleted` is added to the Event History when the Worker confirms that the Update has finished.
```

Comment:
Since this is under "notes to selves": please be aware that we don't mention possible future updates in docs. So we avoid "currently" and "in a future release".

Reply:
Also https://t.mp/edu-request for ticketing.